.github/workflows/check.yml

```diff
@@ -57,6 +57,16 @@ jobs:
       - name: Run Tests
         run: bundle exec fastlane ${{ matrix.lane }}
       
+      - name: Check Static Analyzer Output
+        run:  |
+          if [[ -z `find clang -name "*.html"` ]]; then
+            echo "Static Analyzer found no issues. Fail for demo purpose."
+            exit 123
```

Comment:
We're going to remove this line, clearly, before this PR can land.

Reply:
806e5c9ad6c1138bf0e3a98422695348b6574db7